stingray/events.py

```diff
@@ -680,22 +743,24 @@ def to_astropy_timeseries(self):
     def from_astropy_timeseries(ts):
```

Comment:
These are user-faacing methods, so needs docstring. Especially, it'd be useful to record here whether the `astropy.TimeSeries` object needs to fulfil any specific conditions (e.g. what's in the columns, specific header keywords, ...)